SPECS/prism/prism.spec

```diff
@@ -29,35 +28,58 @@ Requires: xfsprogs
 Requires: zstd
 Requires: veritysetup
 Requires: grub2
+%ifarch x86_64
 Requires: grub2-pc
+%endif
 Requires: systemd-ukify
 
+# %description below is sourced from the "Overview" section of the Prism documentation:
+# https://microsoft.github.io/azure-linux-image-tools/imagecustomizer/README.html
+# Please refer to the docs to keep updated.
 %description
-The Image Customizer is a tool that can take an existing generic Azure Linux image 
-and modify it to be suited for particular scenario.
+Prism is a tool that can take an existing generic Azure Linux image and modify it to
+be suited for particular scenario.
 
-The Image Customizer uses chroot (and loopback block devices) to customize the image. 
-This is the same technology used to build the Azure Linux images, along with most other
-Linux distros. This is in contrast to some other image customization tools, like Packer, 
-which customize the image by booting it inside a VM.
+Prism uses chroot (and loopback block devices) to customize the image. This is the same
+technology used to build the Azure Linux images, along with most other Linux distros.
+This is in contrast to some other image customization tools, like Packer, which customize
+the image by booting it inside a VM.
+
+There are a number of advantages and disadvantages to the chroot approach to customizing images.
+
+Advantages:
+
+- Lower overhead, since you don't need to boot up and shutdown the OS.
+- More precision when making changes, since you won't see any side effects that come
+  from the OS running.
+- The image has fewer requirements (e.g. ssh doesn't need to be installed).
+
+Disadvantages:
+
+- Not all Linux tools play nicely when run under chroot.
+  For example, while it is possible to install Kubernetes using Image Customizer,
+  initialization of a Kubernetes cluster node must be done while the OS is running
+  (e.g. using cloud-init).
 
 %prep
 %autosetup -n azure-linux-image-tools-%{version}
 
 %build
 export GOPATH=%{our_gopath}
-cd toolkit/tools/imagecustomizer
-go build -o %{_builddir}/imagecustomizer .
+make -C ./toolkit go-imagecustomizer REBUILD_TOOLS=y
```

Comment:
FYI -- specs are encouraged to use official RPM macros for invoking standard build tools like `make`. Reference: https://docs.fedoraproject.org/en-US/packaging-guidelines/#_parallel_make